server/utils/statemanager/state_manager.go

```diff
@@ -196,21 +214,23 @@ func isPlayerInGroup(playerName string, playersInGroup []*models.Player) bool {
 // SetGameState is a debug method for forcing the gamestate to make UI testing easier.
 func SetGameState(gameStateName string) (*GameStatusResponse, error) {
 	gameState := models.GameState(gameStateName)
+	mockPrompt := []string{"silly", "great", "beluga"}
+	mockPrompts := [][]string{mockPrompt, mockPrompt, mockPrompt}
 	switch currentState := gameState; currentState {
 	case models.Voting:
-		return createGameState("chats", []string{"graisseux", "frere jacques", "pepe le pew"}, gameState)
+		return createGameState("chats", []string{"graisseux", "frere jacques", "pepe le pew"}, mockPrompts, gameState)
 	case models.WaitingForPlayers:
-		return createGameState("not cats", []string{"dog", "cat", "other dog"}, gameState)
+		return createGameState("not cats", []string{"dog", "cat", "other dog"}, mockPrompts, gameState)
```

Comment:
maybe pass in nil where the value is not needed?

Reply:
I like this a lot